packages/backend/src/controllers/v2/QueryController.ts

```diff
@@ -380,4 +381,30 @@ export class QueryController extends BaseController {
             results,
         };
     }
+
+    @Hidden()
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Get('/{queryUuid}/csv')
```

Comment:
I was expecting this to be under /download so we could support more download types(eg. excel, json) without new endpoints.